### ISSUE TITLE
Status

### ISSUE THREAD

**Issue by asomov** [michajlo/otpakka#1]
Hi,
what is the status of the project ? Are you going to continue ? (I am also interested in this project)

**Comment by michajlo** [comment]
I wound up going all in on Akka actors and haven't really put much thought into this project since.  That being said if you have any interest/ideas feel free to make suggestions, submit pull requests, or fork.

**Comment by asomov** [comment]
I would like to try to implement an (Open Source) OTP Erlang project with Akka. I wonder if this project can be a good start. And I wonder if you feel commitment to support the project in the nearest future.

**Comment by michajlo** [comment]
This is probably a good starting point if you want a more direct translation from the OTP behaviours, though it has been a while so I can't say that with too much authority.

The way actors are created can probably be reworked, I was planning on letting usage drive how that evolved, but never got that far.

As far as supporting the project I'm not really sure what I can offer- this was just something I hacked up in my spare time and I've moved on to other things.  If you have any ideas that are compelling enough I may be tempted to pick this back up, but pull requests are probably the way to go.
